app/src/main/java/org/mifos/mobile/ui/about/AboutUsHeader.kt

```diff
@@ -31,6 +34,11 @@ fun AboutUsHeader() {
         Text(
             text = stringResource(id = R.string.app_name),
             style = MaterialTheme.typography.headlineMedium,
+            color = if(AppCompatDelegate.getDefaultNightMode() == 2 || (AppCompatDelegate.getDefaultNightMode() ==- 1 && isSystemInDarkTheme())) {
+                Color.White
+            }else {
+                       Color.Black
+                       },
```

Comment:
maybe we can get away with just the below line 
`color = if(isSystemInDarkTheme())Color.White else Color.Black` ?